### PR TITLE
Adds LightHouse CI

### PR DESCRIPTION
**Pull request by Berkmann18**
It is targetted to evaluate the`./public` folder _for now_.
There's a way to evaluate URLs but I felt it would restrict it to being run only on `master` push events (like the GH Pages CI) but then it would have to be executed when the GHPCI finishes (= merging both) and won't be able to assess PRs *before* they are merged.

I also tried running LH locally to have an idea of the baseline (e.g. to see what metric scores would be realistically achievable for the beta) but it took too long to even warm up LH.

Re #45

### CODE REVIEW

**Comment by Berkmann18** [comment]
Current (as of d3bd55090c23884ed39bc973b2fa2c13ad0485f5) scores on `index` and `dev` respectively:

<details>
  <summary><code>index.html</code></summary>
  LH score
  <u>Average score:</u> <em>48.802721088435376%</em>
</details>

<details>
  <summary><code>dev.html</code></summary>
  LH score
  <u>Average score:</u> <em>49.98639455782313%</em>
</details>

File: .github/lighthouserc.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "ci": {
+    "preset": "lighthouse:recommended",
+    "assert": {
+      "assertions": {
+        "categories:performance": ["error", {"minScore": 0.4}],
```
Comment (misterpeddy) [inline .github/lighthouserc.json:6]: What is this performance measuring? 
How easy would it be to define our own latency metric and plumb it through to light house (from a cursory look at their docs i remember getting the impression that it was possible - is that right)?

Comment (Berkmann18) [reply .github/lighthouserc.json:6]: It's measuring:
- First Contentful Paint (FCP)
- Speed Index
- Time To Interactive
- First Meaningful Paint
- First CPU Idle
- Max Potential First Input Delay

You can see the results in detail in the CI artefact (like [this one](https://github.com/misterpeddy/hands-down/pull/47/checks?check_run_id=569196345)) which can be downloaded when clicking on "Details" of the _"Lighthouse / static-dist-dir (pull_request)"_ check.

It's fairly straight forward to define assertions for specific metrics (e.g. FCP) as shown on: https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/assertions.md#audits.
As for custom metrics, I'm not sure.

Comment (Berkmann18) [reply .github/lighthouserc.json:6]: @misterpeddy Do you want to have more specific performance assertions?

**Comment by Berkmann18** [comment]
@misterpeddy I've added the documentation, let me know if it needs to be tweaked.

File: README.md [hunk 2]
```diff
@@ -44,3 +44,61 @@ Make sure you have branched off of the latest version of master. Keep commits sh
 There is a pre-commit hook that will check for good code style, and fail commits with an error. Run `npm run lint` to find out more.
 
 Please check for Markdown linting issues if you're updating the `README.md` or any `Markdown`.
+
+### Lighthouse
```
Comment (misterpeddy) [inline README.md:48]: Maybe title the section "Performance Metrics & Insights" and simply add a sentence before Local Testing mentioning that we use Lighthouse for metrics and analysis.

File: README.md [hunk 3]
```diff
@@ -44,3 +44,61 @@ Make sure you have branched off of the latest version of master. Keep commits sh
...
+
+#### Local testing
+If you're locally testing the app and want to see how much the metrics changes based on your changes, you should do the following:
```
Comment (misterpeddy) [inline README.md:51]: s/changes/change

**Pull request merged**